File: orchestrator/android/PRESTOplay-Yospace-Java/app/src/main/java/io/streamroot/lumen/delivery/client/samples/orchestrator/exoplayer/SRApplication.java
package io.streamroot.lumen.delivery.client.samples.orchestrator.exoplayer;

import androidx.multidex.MultiDexApplication;

import io.streamroot.lumen.delivery.client.core.LumenDeliveryClient;

public final class SRApplication extends MultiDexApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        LumenDeliveryClient.initializeApp(this);
    }
}

File: orchestrator/android/ExoPlayer-Yospace/app/src/main/java/io/streamroot/lumen/delivery/client/samples/orchestrator/exoplayer/SRApplication.kt
package io.streamroot.lumen.delivery.client.samples.orchestrator.exoplayer

import androidx.multidex.MultiDexApplication
import io.streamroot.lumen.delivery.client.core.LumenDeliveryClient

class SRApplication: MultiDexApplication() {

    override fun onCreate() {
        super.onCreate()
        LumenDeliveryClient.initializeApp(this)
    }
}